recipes/ncurses/ncurses.inc

```diff
@@ -37,6 +37,8 @@ EXTRA_OECONF += "\
 --without-cxx-binding \
 --with-terminfo-dirs=${sysconfdir}/terminfo:${datadir}/terminfo \
 --enable-overwrite \
+--enable-pc-files \
+--with-pkg-config-libdir=${libdir}/pkgconfig \
```

Comment:
Why not inherit pkgconfig.oeclass instead?  Is ncurses special in this regard, or do you think we need general improvements to pkgconfig.oeclass?